time timestamp  # Time at which message is sent
string topic    # Topic on which message is sent
string message  # Message
